report_thesis/src/sections/background/preprocessing/max_abs.tex

```diff
@@ -2,9 +2,11 @@ \subsubsection{Max Absolute Scaler}
 Max absolute scaling is a normalization technique that scales each feature individually so that the maximum absolute value of each feature is 1.
 This results in the data being normalized to a range between -1 and 1.
 The formula for max absolute scaling is given by:
+
 $$
-	X_{\text{scaled}} = \frac{x}{\max(|x|)},
+x'_i = \frac{x_i}{\max(|\mathbf{x}|)},
 $$
-where $x$ is the original feature value and $X_{\text{scaled}}$ is the normalized feature value.
+
+where $x_i$ is the original feature value, $\max(|\mathbf{x}|)$ is the maximum absolute value of the feature vector $\mathbf{x}$, and $x'_i$ is the normalized feature value.
 This scaling method is useful for data that has been centered at zero or data that is sparse, as max absolute scaling does not center the data.
```

Comment:
You've addressed that it is useful for data that has been centered, because it does not center the data, but I feel like you are missing the point about why its useful for sparse data. IIRC its because it doesnt introduce values in places where there is a 0.